bundle/metadata/annotations.yaml

```diff
@@ -4,12 +4,13 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: nvidia-network-operator
-  operators.operatorframework.io.bundle.channels.v1: 23.10.0-1
-  operators.operatorframework.io.bundle.channel.default.v1: 23.10.0-1
+  operators.operatorframework.io.bundle.channels.v1: v24.7,stable
+  operators.operatorframework.io.bundle.channel.default.v1: v24.7
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.33.0
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
 
   # Annotations for testing.
   operators.operatorframework.io.test.mediatype.v1: scorecard+v1
   operators.operatorframework.io.test.config.v1: tests/scorecard/
+  com.redhat.openshift.versions: v4.9-v4.16
```

Comment:
@rollandf  min OCP should be aligned with our support matrix for network operator 24.7.0 which is min 4.14

Reply:
Done